helm_deploy/wmt-web/values.yaml

```diff
@@ -26,7 +26,7 @@ generic-service:
     NODE_ENV: "production"
     WMT_DB_NAME: "hmpps_workload"
     WMT_DB_ARCHIVE_NAME: "hmpps_workload_archive"
-    WMT_WEB_AUTHENTICATION_ENABLED: "true"
+#    WMT_WEB_AUTHENTICATION_ENABLED: "true"
```

Comment:
wasn't sure if we wanted to keep this as a placeholder for when we do get to configuring auth or remove all together and start again when we do auth?